### PR TITLE
Decorate OTA views with @tracer.wrap to enable tracing

### PR DESCRIPTION
**Pull request by AmitPhulera**
https://dimagi.atlassian.net/browse/SAAS-15465

## Product Description

This PR tags the traces from OTA views with a name and separate service `hq_traces` that can be used to filter the traces on datadog.

## Feature Flag

NA
## Safety Assurance
Should be pretty safe. Just adds tracing. This branch is also added on staging, so we should be able to see these traces for staging as well.
### Safety story

Ran HQ locally after running the changes
### Automated test coverage
NA

### QA Plan
NA

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

**Review by snopoke — DISMISSED** [review]

File: corehq/apps/ota/views.py [hunk 1]
```diff
@@ -83,6 +84,7 @@
 PROFILE_LIMIT = int(PROFILE_LIMIT) if PROFILE_LIMIT is not None else 1
 
 
+@tracer.wrap(name="restore", service='hqtraces')
```
[the remark this answers is not in the document]
Comment (AmitPhulera) [reply corehq/apps/ota/views.py]: https://github.com/dimagi/commcare-hq/pull/34442/commits/bde28720df68264629e18498be7559c23e7bb56a

**Comment by dannyroberts** [comment]
one test job failed, but wasn't obvious why. Restarting it to see if it's just flakey.

**Comment by dannyroberts** [comment]
That didn't fix it

**Comment by dannyroberts** [comment]
Ethan restarted the job again and that fixed it

**Review by dannyroberts — APPROVED** [review]

**Review by ajeety4 — APPROVED** [review]

**Pull request merged**